### ISSUE TITLE
can't change iron-dropdown position:fixed to position:static using --paper-menu-button-dropdown

### ISSUE THREAD

**Issue by davidmaxwaterman** [PolymerElements/paper-menu-button#47]
The source :

<https://github.com/PolymerElements/paper-menu-button/blob/master/paper-menu-button.html#L93>

seems to suggest I can say 'position: static' in --paper-menu-button-dropdown and it would apply to the iron-dropdown, but it doesn't work.

I have this hierarchy :

```
    <paper-toolbar id="toolbar" class="toolbar-shadow">
      <paper-dropdown-menu id="menu" label="All" no-label-float>
        <paper-listbox class="dropdown-content">
          <paper-item>All</paper-item>
          <paper-item>Some</paper-item>
          <paper-item>None</paper-item>
        </paper-listbox>
      </paper-dropdown-menu>
    </paper-toolbar>
```

and the iron-dropdown is inside paper-dropdownmenu. If I try to have the following in the style for the above html :

```
    #menu {
      --paper-menu-button-dropdown: {
        position: maxmaxmax;
      }
    }
```

'maxmaxmax' shows up here :

```
<style scope="paper-menu-button-0">
:host([disabled]) {
  color:#bdbdbd;
}

paper-material {
  background-color:#ffffff;position:maxmaxmax;
}</style>
```

and iron-dropdown still has position: fixed.

What am I missing?

Ref: http://plnkr.co/edit/ULHQ3r?p=preview

**Comment by cdata** [comment]
Hi, thanks for reporting. `position: fixed` is a currently a requirement / limitation of how our overlays are implemented. We are exploring alternative approaches for overlays that may improve this situation, but for now I don't think there is anything actionable that we can do about this. Please re-open as a feature request if you have a specific design in mind!

**Comment by davidmaxwaterman** [comment]
OK, but that's just one of the issues. Why wasn't I get the following?

```
iron-dropdown {
  position:maxmaxmax;
}
```

Well, that's what I was wondering to myself, but when I try the plnkr I made (and change the `position:fixed` to `position:maxmaxmax` in cp-panel.html, then search for `maxmaxmax` in the chrome dev tools' elements pane), I see that, indeed it *is* how I expect. I can only suppose that something is changed in polymer/element or I made some mistake...

Shrug - anyway, all good :)
